Formula/h/hashpump.rb

```diff
@@ -1,7 +1,7 @@
 class Hashpump < Formula
   desc "Tool to exploit hash length extension attack"
-  homepage "https://github.com/bwall/HashPump"
-  url "https://github.com/bwall/HashPump/archive/refs/tags/v1.2.0.tar.gz"
+  homepage "https://archive.softwareheritage.org/browse/origin/directory/?origin_url=https://github.com/bwall/HashPump"
+  url "https://web.archive.org/web/20201018005211/https://github.com/bwall/HashPump/archive/v1.2.0.tar.gz"
```

Comment:
Probably need to update audit to be anchored as this `/ref/tags` was not archived.

Reply:
It's been deprecated for nearly a year and the minimum deprecation period for popular formulae (which this isn't even) is 6 months. We can probably disable this.